.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  test-update-contributors:
```

Comment:
This job does not test "update contributors". It tests the script. Rename the file "test-script.yml", the workflow "Test the gen_dataset script", and the job "test-script".